web/business-registry-dashboard/app/enums/authorized-actions.ts

```diff
@@ -4,12 +4,12 @@
  */
 export enum AuthorizedActions {
   ADD_ENTITY_NO_AUTHENTICATION,
-  HANDLE_BUSINESSES,
-  HANDLE_NAME_REQUESTS,
-  HANDLE_SOCIETIES,
-  HANDLE_TEMPORARY_BUSINESSES,
+  MANAGE_BUSINESS,
+  MANAGE_NR,
+  MANAGE_SOCIETY,
+  RESUME_DRAFT,
   MANAGE_OTHER_ORGANIZATION,
   RESTORATION_REINSTATEMENT_FILING,
-  SEARCH_BUSINESSES,
+  SEARCH_ENTITIES,
```

Comment:
A NR isn't an entity. I don't mind SEARCH_BUSINESS_NR or maybe you have a better idea :)